src/shared/config/route-paths.ts

```diff
@@ -0,0 +1,13 @@
+export const PAGE_PATH = {
+  root: '/',
+  login: '/login',
+  register: '/register',
+  profile: '/profile',
+  article: {
```

Comment:
we had an agreement that you should create a function which will set params in urls and here we can leave only simple URLs]
```
article: '/article/:url'
```